TODO.md

```diff
@@ -1,2 +1,6 @@
-- make conode.go so that it works with the environment variables
-- fix status so `status --host https://test.com:7771` works
+- fix double sign up
```

Comment:
shouldn't these be issues?

Reply:
If I work alone on a project I usually add a TODO and write things down there. It is removed now.